update themes structure, change quality property to brightness

igor-lid opened the pull request:
Issue - #661 
- Old themes array replaced with new themes (https://github.com/ethereum/remix-project/issues/661#issuecomment-739964263), added new values
- quality property is changed to brightness

Please check if values in the theme object are correct.

LianaHus commented:
what is the reason of changing quality to brightness?

LianaHus commented on apps/remix-ide/src/app/tabs/settings-tab.js:
```diff
@@ -59,7 +59,7 @@ module.exports = class SettingsTab extends ViewPlugin {
           const el = yo`<div class="radio custom-control custom-radio mb-1 form-check ${css.crow}">
           <input type="radio" onchange=${event => { onswitchTheme(event, aTheme.name) }} class="align-middle custom-control-input" name="theme" id="${aTheme.name}" data-id="settingsTabTheme${aTheme.name}">
-          <label class="form-check-label custom-control-label" data-id="settingsTabThemeLabel${aTheme.name}" for="${aTheme.name}">${aTheme.name} (${aTheme.quality})</label>
+          <label class="form-check-label custom-control-label" data-id="settingsTabThemeLabel${aTheme.name}" for="${aTheme.name}">${aTheme.brightness[0].toUpperCase() + aTheme.brightness.slice(1)} (${aTheme.brightness})</label>
```
this will not work for other themes and the less readable

igor-lid commented on apps/remix-ide/src/app/tabs/theme-module.js:
```diff
@@ -4,18 +4,63 @@ import QueryParams from '../../lib/query-params'
+    warnContrast: '#212529',
+    error: '#b84040',
+    errorContrast: '#fff',
+    disabled: ''
```
What do you think about these values? These values are active when input is disabled in Remix-ide
light theme - `#e9ecef`
dark theme - `#282a3d`

GrandSchtroumpf replied:
Check with @Liana for the colors if possible.

igor-lid commented:
- added new properties `name` and `disabledContrast` to the theme object
- changed `brightness` to `quality`
- revert code to use `quality`
- added values for `disabled`, `disabledContrast` in `light` and `dark` theme object

LianaHus commented on apps/remix-ide/src/app/tabs/theme-module.js:
```diff
@@ -33,7 +78,6 @@ export class ThemeModule extends Plugin {
       config: registry.get('config').api
     }
     this.themes = themes.reduce((acc, theme) => {
-      theme.url = window.location.origin + window.location.pathname + theme.url
```
@yann300

yann300 replied:
I think we still need that one

LianaHus replied:
@igor-lid

igor-lid replied:
I remove this line because I thought the CDN link is going to be used to load the theme css. e.g. ` url: 'https://res.cloudinary.com/lianahus/raw/upload/v1597918237/remix-themes/PR365/remix-dark_tvx1s2.css'`

Or is it better to rollback to previous logic where the theme css is imported from the assets folder? e.g.  `url: 'assets/css/themes/remix-dark_tvx1s2.css'`
